src/main.cpp

```diff
@@ -169,6 +181,15 @@ void setup()
 {
   Serial.begin(115200);
   WiFi.begin(WIFI_SSID, WIFI_PASSKEY);
+
+  // Set up the servo
+  ESP32_ISR_Servos.useTimer(1);
+  servoIndex = ESP32_ISR_Servos.setupServo(0, 500, 2500);
+  if (servoIndex == -1) {
+    Serial.println("Servo could not start!");
```

Comment:
This seems catastrophic enough to warrant setting an error flag and flashing the LED red or something